### PR TITLE
Chat: make day header's font-size to be equal 12px in all themes and modes

### PR DESCRIPTION
**Pull request by ksercs**
…

### CODE REVIEW

File: e2e/testcafe-devextreme/tests/chat/etalons/Messagegroup appearance in RTL mode (material-blue-light).png
Comment (ksercs) [inline e2e/testcafe-devextreme/tests/chat/etalons/Messagegroup appearance in RTL mode (material-blue-light).png:1]: to replace

File: e2e/testcafe-devextreme/tests/chat/etalons/Typing indicator with a lot of items (fluent-blue-light).png
Comment (ksercs) [inline e2e/testcafe-devextreme/tests/chat/etalons/Typing indicator with a lot of items (fluent-blue-light).png:1]: to replace

Comment (ksercs) [reply e2e/testcafe-devextreme/tests/chat/etalons/Typing indicator with a lot of items (fluent-blue-light).png:1]: https://github.com/DevExpress/DevExtreme/pull/28440/commits/f8c6536a54051c7349704936a7a4d8a2bba42eb4

File: e2e/testcafe-devextreme/tests/chat/etalons/Messagegroup appearance in RTL mode (material-blue-light).png
Comment (ksercs) [reply e2e/testcafe-devextreme/tests/chat/etalons/Messagegroup appearance in RTL mode (material-blue-light).png:1]: Its produced this etalon 3 times, looks like we need to merge it with this appearance

Comment (ksercs) [reply e2e/testcafe-devextreme/tests/chat/etalons/Messagegroup appearance in RTL mode (material-blue-light).png:1]: https://github.com/DevExpress/DevExtreme/pull/28440/commits/12194c6b9229261c672b3e59e091712396dfd63d

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request merged**